sentences/nl/light_HassTurnOn.yaml

```diff
@@ -9,6 +9,8 @@ intents:
           - "Schakel [alle] <lamp> [in] <area> in"
           - "<doe> (alle | in) <area> <lamp> aan"
           - "Schakel (alle | in) <area> <lamp> in"
+          - "<lamp> aan"
+          - "<lamp> aan in <area>"
```

Comment:
You can make `<doe>` optional in `"<doe> [alle] <lamp> [<naar>] aan in <area>"` to make this option possible without adding an additional sentence.

Reply:
Makes sense, I've changed that.